commit a98f9e8c85c06d71f77210960609c228f9372d71
Author:     Dongkyun Son <dongkyun.s@samsung.com>
AuthorDate: Fri Oct 16 13:50:47 2026 +0900
Commit:     Dongkyun Son <dongkyun.s@samsung.com>
CommitDate: Fri Oct 16 13:51:20 2026 +0900

    Revert "Enable DOD path for Tizen-Base-X project_meta"
    
    This reverts commit 16c83914bdd7f6f65610091a9c4c89f0ec580fa7.
    
    Change-Id: Iba61a6a552cb3efea8a7d55a32735dddc8a0aad4
---
 TIZEN/Tizen/Tizen-Base-X/project_meta | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TIZEN/Tizen/Tizen-Base-X/project_meta b/TIZEN/Tizen/Tizen-Base-X/project_meta
index 406b4de..f7944ec 100644
--- a/TIZEN/Tizen/Tizen-Base-X/project_meta
+++ b/TIZEN/Tizen/Tizen-Base-X/project_meta
@@ -3,7 +3,7 @@
     <enable/>
   </debuginfo>
   <repository name="standard">
-    <path project="DOD:Tizen-Base-X" repository="standard" />
+    <!--path project="DOD:Tizen-Base-X" repository="standard" /-->
     <arch name="armv7l"/>
     <arch name="aarch64"/>
     <arch name="x86_64"/>
